src/components/PrivacyStatement/style.css

```diff
@@ -15,72 +15,69 @@
   display: flex;
   justify-content: space-between;
   flex-direction: row;
-  margin-bottom: 24px;
+  margin-bottom: 24*@unit;
   @media (--small-viewport) {
     margin-bottom: 0;
   }
 }
 
 .list {
-  font-size: 16px;
-  line-height: 24px;
+  line-height: 1.5;
```

Comment:
this is a quite common, I wonder if we should make it as past of a text base mixin, what do you think?

Reply:
How do you mean? `line-height: 1.5` you mean?
I think we have quite a few different line-heights that we use

Reply:
yep, `line-height: 1.5` is used in 5 places, hence why I wonder if it could be part of a standard text base. but no strong feelings about it tbh